linkerd/hatch.toml

```diff
@@ -0,0 +1,12 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["27", "38"]
+
+[envs.default.env-vars]
+DDEV_SKIP_GENERIC_TAGS_CHECK = "true"
+
+[envs.default]
+dependencies = [
+  "requests-mock==1.4.0",
```

Comment:
I only see this dependency in the tests, does it need to be shipped with the final package as well?

Reply:
No